discussment-core/src/main/java/org/danekja/discussment/core/service/imp/DefaultDiscussionService.java

```diff
@@ -63,6 +66,16 @@ public Discussion getDiscussionById(long discussionId) throws AccessDeniedExcept
         }
     }
 
+    public Discussion getDefaultDiscussion() {
+        Discussion d = discussionDao.getById(Discussion.DEFAULT_DISCUSSION_ID);
+        if(d == null){
+            d =  new Discussion("default discussion", null);
```

Comment:
Doesnt set Discussion ID to Discussion.DEFAULT_DISCUSSION_ID. May work only by coincidence.